### DISCUSSION TITLE
App Development

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#36444] [Mobile]
Hey everyone,

Can someone please suggest that how can I start with app development for both Android and iOS?

**Comment by iskae** [comment]
Hello @iLakshya 👋 

Thanks for being a part of the GitHub Community! We're glad you're here.

Unfortunately the GitHub Community on Discussions focuses primarily on GitHub itself, therefore it might not be the right place for this particular topic 😅 

We would love to support you if you have any feedback on the GitHub Mobile Apps 🙇 

Cheers!

**Reply by monsiu** [reply]
Hey i would suggest flutter as a start...its not that hard to get into and you build for ios , android and linux plus web simultaneously under the flutter framework!

I also started creating an app like 3 days ago and its going quite well 😁

**Comment by szarath** [comment]
.net Maui or use plain old xamarin on its own
